testutils/call.go

```diff
@@ -95,8 +96,14 @@ func NewIncomingCall(callerName string) tchannel.IncomingCall {
 	return &FakeIncomingCall{CallerNameF: callerName}
 }
 
+// KeyVal is a pair of keyword and values which can be appended to arg2
+type KeyVal struct {
+	Key, Val []byte
```

Comment:
should this just be `string`? we use `[]byte` for performance but the test versions just store strings, see below for service/method/caller

Reply:
`CallFrame.Arg2Append()` accepts `[]byte`, so we'd have to convert the inputs to strings when storing them. probably a bit redundant?

Reply:
True, but since it's a test-only helper, and we are often dealing with strings, I think that should be ok.